data/json/items/comestibles/other.json

```
@@ -70,17 +70,15 @@
     "type": "COMESTIBLE",
     "id": "royal_jelly",
     "name": { "str": "royal jelly", "str_pl": "royal jellies" },
-    "category": "drugs",
+    "category": "food",
     "weight": "150 g",
     "color": "white",
-    "use_action": "ROYAL_JELLY",
     "comestible_type": "FOOD",
     "symbol": "%",
-    "healthy": 10,
+    "healthy": 1,
     "calories": 240,
-    "description": "A translucent hexagonal chunk of wax, filled with dense, milky jelly.  Delicious, and rich with the most beneficial substances the hive can produce, it is useful for curing all sorts of afflictions.",
+    "description": "A translucent hexagonal chunk of wax, filled with dense, milky jelly.  Delicious, and rich with the most beneficial substances the hive can produce, contrary to some belief it doesn't cure anything.",
```

```suggestion
    "description": "A translucent hexagonal chunk of wax, filled with dense, milky jelly.  Contrary to some belief it doesn't cure anything.   Still, it is delicious and rich with the most beneficial substances the hive can produce.",
```

Last sentence seemed awkward, though I dunno if this is much of an improvement.

---

```suggestion
    "description": "A translucent hexagonal chunk of wax, filled with dense, milky jelly.  Though some hold it as a panacea, it doesn't have any medical benefit.  Still, it is delicious, and rich with the most beneficial substances the hive can produce..",
```
Still a bit awkward and clashes a tiny bit with the beneficial substances bit.